penrose-web/src/shapes/mathtransform/mathtest.ts

```diff
@@ -0,0 +1,95 @@
+import { canvasSize } from "./Canvas";
```

Comment:
Why is this file in the `shapes` folder? It's probably a good idea to put all test source files (usually `json` for us, but these `ts` files in your case) in `__tests__` for now. I think we have it configured such that the TS compiler will not try to typecheck files in `__tests__`.

Reply:
Done in latest commit.